server/yarn.lock

```diff
@@ -5505,6 +5613,13 @@ source-map@^0.7.3:
   resolved "https://registry.yarnpkg.com/source-map/-/source-map-0.7.3.tgz#5302f8169031735226544092e64981f751750383"
   integrity sha512-CkCj6giN3S+n9qrYiBTX5gystlENnRW5jZeNLHpe6aue+SrHcG5VYwujhW9s4dY31mEGsxBDrHR6oI69fTXsaQ==
 
+sparse-bitfield@^3.0.3:
+  version "3.0.3"
+  resolved "https://registry.yarnpkg.com/sparse-bitfield/-/sparse-bitfield-3.0.3.tgz#ff4ae6e68656056ba4b3e792ab3334d38273ca11"
+  integrity sha1-/0rm5oZWBWuks+eSqzM004JzyhE=
+  dependencies:
+    memory-pager "^1.0.2"
+
 spdx-correct@^3.0.0:
   version "3.1.1"
```

Comment:
Considering we haven't really implemented much yet, the detail in package and yarn files are head spinning. :/

Reply:
Don't worry about the `.lock` files - these are just to ensure that all dependencies (and dependencies of our dependencies) use the exact same versions when installing the application - you can just ignore the `.lock` files when reviewing

Reply:
Can't ignore them when they cause merge conflicts :)

Reply:
Yeah that was my bad - i initially set up dependencies that should've been in `devDependencies`